screens/Request/RequestLayout.tsx

```diff
@@ -56,6 +76,30 @@ export const RequestLayout: React.FC = () => {
           component={RequestScreen}
           options={{
             title: t('receiveCard').toUpperCase(),
+            headerLeft: () =>
+              I18nManager.isRTL ? (
+                <></>
+              ) : (
+                <HeaderBackButton
+                  onPress={() => {
+                    controller.GOTO_HOME();
+                  }}
+                  style={Theme.Styles.IconContainer}
+                  tintColor={Theme.Colors.Icon}
+                />
+              ),
+            headerRight: () =>
+              I18nManager.isRTL ? (
```

Comment:
same here, can we try this instead of having empty view?
`I18nManager.isRTL &&  <HeaderBackButton onPress={() => { controller.GOTO_HOME(); }} style={Theme.Styles.IconContainer} tintColor={Theme.Colors.Icon} />`

Reply:
updated the condition as per suggested